options/options.cc

```diff
@@ -122,6 +123,9 @@ void ColumnFamilyOptions::Dump(Logger* log) const {
   ROCKS_LOG_HEADER(
       log, "       Options.compaction_filter_factory: %s",
       compaction_filter_factory ? compaction_filter_factory->Name() : "None");
+  ROCKS_LOG_HEADER(
+      log, "       Options.sst_partitioner_factory: %s",
```

Comment:
nit: the log line should aligned at the semicolon.

Reply:
Fixed. I have kept the other that were not aligned like compaction_filter_factory to not cause some regression.